wandb/apis/public/runs.py

```diff
@@ -1498,5 +1503,17 @@ def _beta_scan_history(
             max_step=max_step or self.lastHistoryStep + 1,
             keys=keys,
             page_size=page_size,
+            use_cache=use_cache,
         )
         return beta_history_scan
+
+    def purge_run_history_cache(self):
+        """Purges the run history cache for the run."""
+        cache_dir = env.get_cache_dir()
+        run_history_cache_dir = cache_dir / "runhistory"
```

Comment:
I'd add a higher-level purge cash command to the cli, following a pattern that would be familiar to the user (say, docker prune or pip cache purge). specifics such as what exactly to purge could then be flags of that command (again, feel free to draw inspiration from docker or pip or similar).

It's a bit weird that you'd need to get an instance of an api run to only then be able to purge the cache of its history pq files.

Prob best for that to be its own PR, this one is about using cache.